pytripgui/contouring_vc/contouring_view.py

```diff
@@ -46,11 +48,11 @@ def update_accepted(self):
         self._ui.button_box.button(QDialogButtonBox.Ok).clicked.connect(self._ui.accept)
 
     def update_progress(self, voi_name, current, total):
-        self._set_progress_label_text(self._progress_message.format(name=voi_name, current=current+1, total=total))
+        self._set_progress_label_text(self._progress_message.format(name=voi_name, current=current + 1, total=total))
 
-    def update_finished(self):
+    def update_finished(self, time):
```

Comment:
Not a good idea to name variable `time` - it will shadow `time` module.
Better `time_s` or something more descriptive